tests/apps/courses/test_admin_course_run.py

```diff
@@ -95,22 +95,33 @@ def test_admin_course_run_list_view_superuser(self):
         response = self.client.get(url, follow=True)
         self.assertEqual(response.status_code, 200)
 
-        # Check that the page includes only the draft course run
+        # Check that only the id field is displayed
+        self.assertContains(response, "field-id", 2)
```

Comment:
I also add `self.assertContains(response, "field-", 2)`
In this way we have the guarantee that two `field-id` are present on the page and there is only this two fields.

Reply:
Oh yeah... right.

<img src="https://media.giphy.com/media/zQUylKLuRF4kg/giphy.gif">